e2e/infrastructure/TransactionHttp.spec.ts

```diff
@@ -197,6 +205,118 @@ describe('TransactionHttp', () => {
+                mosaicId,
+                new MosaicId(UInt64.fromUint(0).toDTO()),
+                UInt64.fromUint(60641),
+                UInt64.fromUint(0),
+                MosaicRestrictionType.NONE,
+                UInt64.fromUint(0),
+                MosaicRestrictionType.GE,
+                NetworkType.MIJIN_TEST,
+            );
+            const signedTransaction = mosaicGlobalRestrictionTransaction.signWith(account, generationHash);
+
+            listener.confirmed(account.address).subscribe((transaction: Transaction) => {
+                done();
+            });
+            listener.status(account.address).subscribe((error) => {
+                console.log('Error:', error);
+                assert(false);
+                done();
+            });
+            transactionHttp.announce(signedTransaction);
+        });
+    });
+    describe('MosaicGlobalRestrictionTransaction', () => {
+        let listener: Listener;
+        before (() => {
+            listener = new Listener(config.apiUrl);
+            return listener.open();
+        });
+        after(() => {
+            return listener.close();
+        });
+        it('aggregate', (done) => {
+            const mosaicGlobalRestrictionTransaction = MosaicGlobalRestrictionTransaction.create(
+                Deadline.create(),
+                mosaicId,
+                new MosaicId(UInt64.fromUint(0).toDTO()),
+                UInt64.fromUint(60641),
+                UInt64.fromUint(0),
+                MosaicRestrictionType.GE,
+                UInt64.fromUint(1),
+                MosaicRestrictionType.GE,
+                NetworkType.MIJIN_TEST,
+            );
+            const aggregateTransaction = AggregateTransaction.createComplete(Deadline.create(),
+                [mosaicGlobalRestrictionTransaction.toAggregate(account.publicAccount)],
+                NetworkType.MIJIN_TEST,
+                [],
+            );
+            const signedTransaction = aggregateTransaction.signWith(account, generationHash);
+            listener.confirmed(account.address).subscribe((transaction: Transaction) => {
+                done();
+            });
+            listener.status(account.address).subscribe((error) => {
+                console.log('Error:', error);
+                assert(false);
+                done();
+            });
+            transactionHttp.announce(signedTransaction);
+        });
+    });
+
+    describe('MosaicAddressRestrictionTransaction', () => {
+        let listener: Listener;
+        before (() => {
+            listener = new Listener(config.apiUrl);
+            return listener.open();
+        });
+        after(() => {
+            return listener.close();
+        });
+        it('aggregate', (done) => {
+            const mosaicAddressRestrictionTransaction = MosaicAddressRestrictionTransaction.create(
+                Deadline.create(),
+                mosaicId,
+                UInt64.fromUint(60641),
```

Comment:
maybe add a comment to say "why 60641" ? maybe breaking it in its sub part :+1:

Reply:
hmm, 60641 has got no meaning. It is just a random number I used here for the test.  Does the value needs to be build from 'something'?

Reply:
ah ok , I understood that it was a composed value :+1: